source/content/guides/drupal/drupal-hosted-createbt/01-introduction.md

```diff
@@ -1,31 +1,31 @@
 ---
-title: Migrate a Site That Was Created with Build Tools to Drupal 9
+title: Migrate a Site That Was Created with Build Tools to the Latest Version of Drupal
 subtitle: Introduction
-description: Learn how to migrate a Drupal 8 Site to Drupal 9
-cms: "Drupal 9"
+description: Learn how to migrate a Drupal 8 Site to the latest version of Drupal
```

Comment:
@stovak , should it be "Learn how to update a site to the latest version of Drupal" (omitting ref to D8)?